README.md

```diff
@@ -31,7 +31,7 @@ FarmData2 is powered by the [farmOS](https://not.the.right.link2) open source pr
 Support and assistance with FarmData2 development has been received from [The Non-Profit FOSS Institute](https://npfi.org/).
 
 The development of FarmData2 has received partial support from:
-* The [GNOME Community Engagement Challenge](https://not.the.right.link3):
+* [The [GNOME Community Engagement Challenge](https://not.the.right.link3)](https://www.gnome.org/challenge/):
```

Comment:
Hi @JeremiahAHoward ! It's not necessary that you change it (I know you moved onto the merge conflict practice portion of this activity) - but just want to point out that you added the link _next_ to the "broken" one, instead of replacing it :) Just something to watch for in the future when you're more comfortable with git workflow ✨